Source/player.cpp

```diff
@@ -916,12 +916,10 @@ bool PlrHitMonst(int pnum, int m)
 	}
 
 	if ((player._pIFlags & ISPL_FIREDAM) != 0 && (player._pIFlags & ISPL_LIGHTDAM) != 0) {
-		int midam = player._pIFMinDam + GenerateRnd(player._pIFMaxDam - player._pIFMinDam);
+		int midam = player._pIFDamage.GetValue();

```

Comment:
This isn't equivalent (old code was `rand(max - min)`, this is `rand(max-min+1)`. The old code looked like an error anyway, does `_pIFDamage.maxValue` need to be adjusted to compensate or are we fine with this change?

Reply:
Check commit history. There is one commit that specifically fixes this. I know it's not equivalent, but that was on purpose.

Reply:
Ahh, I should've checked that (once github let me). Thanks for clarifying :)